odoo/addons/base/migrations/13.0.1.3/pre-migration.py

```diff
@@ -463,9 +465,16 @@
 ]
 
 column_renames = {
+    'ir_attachment': [
+        ('name', None),
+        ('datas_fname', 'name'),
+    ],
     'res_lang': [
         ('week_start', None),
     ],
+    'ir_server_object_lines': [
+        ('type', 'evaluation_type'),
+    ],
```

Comment:
I think better use openupgrade.rename_fields instead of openupgrade.rename_columns in order to keep translation

Reply:
Agreed! Code adapted. I also removed the use_env=True argument to the migrate decorator as it has been the default for some time now.